dtls.c

```diff
@@ -1539,6 +1510,98 @@ dtls_prepare_record(dtls_peer_t *peer, dtls_security_parameters_t *security,
   return 0;
 }
 
+/**
+ * Send Alert in stateless fashion.
+ * A Alert is sent to the peer (using the write callback function
+ * registered with \p ctx). The return value is \c -1 on error, \c 1 when
+ * the Alert is sent.
+ *
+ * \param ctx              The DTLS context.
+ * \param ephemeral_peer   The ephemeral remote party we are talking to.
+ * \param level            Alert level.
+ * \param description      Alert description.
+ * \return \c 1 if msg is sent, or \c -1 otherwise.
```

Comment:
Actually, the result is "a value less than zero" on error (as stated in the documentation of the `write` callback for `dtls_handler_t`) or "the number of bytes sent".

Reply:
Ok